internal/uvm/start.go

```diff
@@ -58,11 +55,36 @@ func forwardGcsLogs(l net.Listener) {
 	}
 }
 
+func printToStdout(c net.Conn) {
+	io.Copy(os.Stdout, c)
+}
+
+func processOutput(l net.Listener, doneChan chan struct{}, handler outputHandler) {
+	defer close(doneChan)
+
+	c, err := l.Accept()
+	l.Close()
+	if err != nil {
+		logrus.Error("accepting log socket: ", err)
+		return
+	}
+	defer c.Close()
+
+	handler(c)
+}
+
 // Start synchronously starts the utility VM.
 func (uvm *UtilityVM) Start() error {
-	if uvm.gcslog != nil {
-		go forwardGcsLogs(uvm.gcslog)
-		uvm.gcslog = nil
+	if uvm.outputListener != nil {
+		var handler outputHandler
+		switch uvm.outputHandling {
```

Comment:
>outputHandling [](start = 13, length = 14)

should the options struct just contain a handler so that uvmboot can provide the printToStdout handler itself?